addons/post_processing/node/post_process.gd

```diff
@@ -119,7 +119,7 @@ func _check_shader_visibility(_name: String) -> bool:
 
 		if _name.begins_with("CRT"):
 			return true if configuration.CRT else false
-		
+		# get_children() returning all _names leading Always to:
```

Comment:
The error is being triggered every reload due to the get_children() method returning more than expected.